nextflow/Remapping/modules/copy.nf

```diff
@@ -15,10 +16,16 @@ process copy_to_rapid_release_ftp {
 
   script:
     def assembly = !params.keep_id && lookup[id] ? lookup[id] : id
+    def outdir = params.rr_root + "/" + params.rr_path
   """
   assembly=${assembly}
   for file in ${vcf}; do
-    become ensrapid rsync -av `realpath \${file}` ${params.rr_root}/${params.rr_path}
+    become ensrapid rsync -av `realpath \${file}` ${outdir}
   done
+
+  # Add or update checksums
+  become ensrapid rm -f ${outdir}/md5sum* ${outdir}/CHECKSUMS
+  become ensrapid \
+    bash -c 'assembly=${assembly} && cd ${outdir} && md5sum * > CHECKSUMS'
```

Comment:
Hi Nakib! We need to redefine `assembly` here or the variable `$assembly` will be undefined when launching a bash process for the `ensrapid` user. I tried `export`ing the variable, but it didn't work. Do you have any idea on how to avoid defining the variable twice?

Reply:
Hi @nuno-agostinho, 

If you need the `$assembly` variable in that shell then I think you are doing is correct.
But I am trying to understand, why you are needing `assembly` variable - you are using `*` glob for md5sum command and `outdir` variable does not seem to need `assembly`.

Reply:
Changed the code to use double-quotes, thus avoiding the need to re-define `$assembly` in the subshell.